schema.graphql

```diff
@@ -70,6 +75,7 @@ type Dataset {
   status: JobStatus
   inputPath: String
   uploadDateTime: String
+  fdrCounts(inpFdrLvls: [Float]): FdrCountsNumber
```

Comment:
I'd use `!` as a non-null specification everywhere in this declaration and definition of `FdrCountsNumber`